trussworks/react-uswds
# feat: upgrade to USWDS v3.1.0

# Summary

This PR installs the new version of USWDS and follows the required steps in the [migration guide](https://designsystem.digital.gov/documentation/migration/).

- Gets rid of `uswdsResources.css` and moves the settings/overrides into `_uswds-theme.css` (as suggested by the guide.)
- Updates Sass entry point to use `@forward` syntax.
- Updates paths to USWDS assets

## Related Issues or PRs

Closes #2199 


## How To Test


Test by running `yarn build`, `yarn test`, and `yarn run storybook`. No errors should appear.



### Screenshots (optional)
